encoding/protobuf/protoc-gen-yarpc-go/internal/lib/lib.go

```diff
@@ -642,6 +661,43 @@ func clientServerStreamingMethods(service *protoplugin.Service) ([]*protoplugin.
 	return methods, nil
 }
 
+func fileDescriptorClosureVarName(f *protoplugin.File) (string, error) {
```

Comment:
Can we add a comment on why we're doing this? It makes sense seeing this PR altogether, but this function on it's own may seem a bit cryptic.